go/cmd/vtctldclient/command/serving_graph.go

```diff
@@ -245,4 +282,13 @@ func init() {
 
 	RebuildVSchemaGraph.Flags().StringSliceVarP(&rebuildVSchemaGraphOptions.Cells, "cells", "c", nil, "Specifies a comma-separated list of cells to look for tablets.")
 	Root.AddCommand(RebuildVSchemaGraph)
+
+	noValueIndicator := "~"
```

Comment:
why not just put the literal `"~"` in the flag definition below?

Reply:
Just to give it a name, which better explain what it means.. I should really add some comments though.

Reply:
added clarifying comment